tests/integration/docker_images/common.bash

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+DOCKER_EXE="docker"
+DOCKER_SERVICE="docker-cor"
+
+#Cleaning test environment
+function cleanDockerPs(){
```

Comment:
This file looks identical to `tests/integration/docker/common.bash`. I think it would make sense to:

- Move `tests/integration/docker/common.bash` to `tests/integration/lib/common.bash`.
- Make the `docker` tests and the `docker_images` tests both use `tests/integration/lib/common.bash`